catboost/python-package/catboost/core.py

```diff
@@ -1589,11 +1589,24 @@ def get_object_importance(self, pool, train_pool, top_size=-1, ostr_type='Averag
             Number of threads.
             If -1, then the number of threads is set to the number of cores.
 
+        verbose : bool or int
+            If False, then evaluation is not logged. If True, then each possible iteration is logged.
```

Comment:
Something like "if N > 0, then add one line to the log after processing each batch of N objects"